demo/BERT/builder.py

```diff
@@ -524,26 +524,26 @@ def load_onnx_weights_and_quant(path, config):
 
 def emb_layernorm(builder, network, config, weights_dict, builder_config, sequence_length, batch_sizes):
     if len(batch_sizes) > 1:
-        input_ids = network.add_input(name="input_ids", dtype=trt.int32, shape=(sequence_length, -1))
-        segment_ids = network.add_input(name="segment_ids", dtype=trt.int32, shape=(sequence_length, -1))
-        input_mask = network.add_input(name="input_mask", dtype=trt.int32, shape=(sequence_length, -1))
+        input_ids = network.add_input(name="input_ids", dtype=trt.int32, shape=(-1, sequence_length))
+        segment_ids = network.add_input(name="segment_ids", dtype=trt.int32, shape=(-1, sequence_length))
+        input_mask = network.add_input(name="input_mask", dtype=trt.int32, shape=(-1, sequence_length))
 
         # Specify profiles for the batch sizes we're interested in.
         # Make sure the profile also works for all sizes not covered by the previous profile.
         prev_size = 0
         for batch_size in sorted(batch_sizes):
             profile = builder.create_optimization_profile()
-            min_shape = (sequence_length, prev_size + 1)
-            shape = (sequence_length, batch_size)
-            profile.set_shape("input_ids", min=min_shape, opt=shape, max=shape)
-            profile.set_shape("segment_ids", min=min_shape, opt=shape, max=shape)
-            profile.set_shape("input_mask", min=min_shape, opt=shape, max=shape)
+            min_shape = (1, sequence_length)
```

Comment:
min_shape = (prev_size + 1， sequence_length) ?

Reply:
You're right, will push it in